source/modulo_core/include/modulo_core/Cell.hpp

```diff
@@ -534,6 +535,13 @@ class Cell : public rclcpp_lifecycle::LifecycleNode {
    * @param period the period between two calls
    */
   void add_periodic_call(const std::function<void(void)>& callback_function, const std::chrono::nanoseconds& period);
+
+  /**
+   * @brief Function to daemonized the callback function given in input
```

Comment:
This and the method above could benefit from a slight elaboration or even a @details tag,
to disambiguate it from the periodic call (that the callback function
is always evaluated, while the periodic call wraps the callback
in the run_periodic_call method to filter for the active state

Also
```suggestion
   * @brief Function to daemonize the callback function given in input
```